### PR TITLE
chromium.py: Update default target module to chrome_public

### PR DESCRIPTION
**Pull request by legendlee1314**
The patch to remove ChromeShell related build target was landed
as [#346997](https://codereview.chromium.org/1317453005).

### CODE REVIEW

**Comment by legendlee1314** [comment]
@gyagp , @YuZhiqiangX , I'm not sure if this patch will block some buildings else. Could you please check?

File: python/chromium.py [hunk 1]
```diff
@@ -327,7 +327,7 @@ def parse_arg():
     group_common.add_argument('--target-os', dest='target_os', help='target os', choices=['android', 'linux'], default='android')
     group_common.add_argument('--target-arch', dest='target_arch', help='target arch', choices=['x86', 'arm', 'x86_64', 'arm64'], default='x86')
     group_common.add_argument('--target-module', dest='target_module', help='target module to build, choices can be chrome, webview_shell, \
-        content_shell, chrome_shell, chromedriver, cpu_features, system_webview_apk, android_webview_telemetry_shell_apk, etc.', default='default')
+        content_shell, chrome_public, chromedriver, cpu_features, system_webview_apk, android_webview_telemetry_shell_apk, etc.', default='default')
```
Comment (gyagp) [inline python/chromium.py]: Don't remove chrome_shell, and just add chrome_public

File: python/chromium.py [hunk 2]
```diff
@@ -308,7 +308,7 @@ def parse_arg():
   python %(prog)s --run --run-option 'http://browsermark.rightware.com'
 
   debug & perf:
-  python %(prog)s --perf --target-module chrome_shell --process render --perf-second 5 --perf-binary-host /usr/bin/perf
+  python %(prog)s --perf --target-module chrome_public --process render --perf-second 5 --perf-binary-host /usr/bin/perf
```
Comment (gyagp) [inline python/chromium.py]: no need to change this

File: python/chromium.py [hunk 3]
```diff
@@ -952,8 +952,8 @@ def build(force=False):
         cmd_ninja += ' android_webview_apk libwebviewchromium'
     elif target_os == 'android' and target_module == 'content_shell':
         cmd_ninja += ' content_shell_apk'
-    elif target_os == 'android' and target_module == 'chrome_shell':
-        cmd_ninja += ' chrome_shell_apk'
+    elif target_os == 'android' and target_module == 'chrome_public':
+        cmd_ninja += ' chrome_public_apk'
```
Comment (gyagp) [inline python/chromium.py]: Chrome public is already added.

**Comment by gyagp** [comment]
We need to keep the build of chrome shell, for old revisions. It's better to just add a check of (rev, target_module) before build.

**Comment by legendlee1314** [comment]
Updated the patch. Please review.

File: python/chromium.py [hunk 4]
```diff
@@ -953,6 +953,9 @@ def build(force=False):
     elif target_os == 'android' and target_module == 'content_shell':
         cmd_ninja += ' content_shell_apk'
     elif target_os == 'android' and target_module == 'chrome_shell':
+        _setup_rev()
+        if rev >= 346997:
+            error('Failed to build ChromeShell after revision 346997')
```
Comment (gyagp) [inline python/chromium.py]: Please change "Failed" to "Can not"

Comment (legendlee1314) [inline python/chromium.py]: Done.

**Pull request merged**